src/pybind/mgr/dashboard/run-backend-api-tests.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-set -eo pipefail
+# set -eo pipefail
```

Comment:
I'm thinking about simply removing this commit form this pr, as it is kind of unrelated.

Reply:
Maybe a PR referencing [[Bug#42402] mgr/dashboard: run-backend-api-tests.sh cannot be sourced anymore](https://tracker.ceph.com/issues/42402) makes sense.

Reply:
@sebastian-philipp I agree.

Reply:
#31070